src/nvim/mouse.c

```diff
@@ -246,6 +246,16 @@ static int get_fpos_of_mouse(pos_T *mpos)
   return IN_BUFFER;
 }
 
+void do_move_autocmd(int screenrow, int screencol) {
+	Dictionary data = ARRAY_DICT_INIT;
+	PUT(data, "screenrow", INTEGER_OBJ(screenrow));
+	PUT(data, "screencol", INTEGER_OBJ(screencol));
+	Object obj = DICTIONARY_OBJ(data);
+	apply_autocmds_group(EVENT_MOUSEMOVED, NULL, NULL, false, AUGROUP_ALL, NULL, NULL, &obj);
+	api_free_dictionary(data);
```

Comment:
Could use input on this - I saw similar free calls used in other files, but not sure if this is actually the proper way to free these values or if freeing the Object and its underlying Dictionary will result in a double free.